README.md

```diff
@@ -187,7 +187,7 @@ Options can be passed either as a data (data-slider-foo) attribute, or as part o
 | handle |	string |	'round' |	handle shape. Accepts: 'round', 'square', 'triangle' or 'custom' |
 | reversed | bool | false | whether or not the slider should be reversed |
 | enabled | bool | true | whether or not the slider is initially enabled |
-| formatter |	function |	returns the plain value |	formatter callback. Return the value wanted to be displayed in the tooltip |
+| formatter |	function |	returns the plain value |	formatter callback. Return the value wanted to be displayed in the tooltip, useful for string values. If a string is returned it will be indicated in an aria-valuetext attribute.  |
```

Comment:
could you wrap the "aria-valuetext" string in backticks (`) so that it stands out visually in the README?

`aria-valuetext`